File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/AutocompleteMetadataResponseModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content;

public class AutocompleteMetadataResponseModel {

	private MetadataInstanceModel metadataInstance;
	private MetadataCollectionInstanceModel metadataCollectionInstance;
	
	public MetadataInstanceModel getMetadataInstance() {
		return metadataInstance;
	}
	public void setMetadataInstance(MetadataInstanceModel metadataInstance) {
		this.metadataInstance = metadataInstance;
	}
	public MetadataCollectionInstanceModel getMetadataCollectionInstance() {
		return metadataCollectionInstance;
	}
	public void setMetadataCollectionInstance(MetadataCollectionInstanceModel metadataCollectionInstance) {
		this.metadataCollectionInstance = metadataCollectionInstance;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/content/autocomplete-metadata-response.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { MetadataInstanceModel } from "./metadata-instance.model";
import { MetadataCollectionInstanceModel } from "./metadata-collection-instance.model";

@JsonObject
export class AutocompleteMetadataResponseModel {
	
	@JsonProperty("metadataInstance", MetadataInstanceModel)
	public metadataInstance: MetadataInstanceModel = null;
	
	@JsonProperty("metadataCollectionInstance", MetadataCollectionInstanceModel)
	public metadataCollectionInstance: MetadataCollectionInstanceModel = null;
}